src/converter/one/util/ResourceProcessor.java

```diff
@@ -0,0 +1,33 @@
+package one.util;
```

Comment:
These are all converter related utils. Why not just put these classes in the `convert` package then?

Reply:
done